update_versions.yml

```diff
@@ -65,6 +65,16 @@
       with_items:
         - "{{ groups.zope|first }}"
 
+- name: get press version
+  hosts: frontend
+  tasks:
+    - shell: grep '^[0-9]' /var/cnx/apps/press/docs/source/changes.rst  | head -n 1
+      register: press_version
```

Comment:
Is there any chance this could conflict with the `press_version` variable?

Reply:
Apparently so, see #636.